src/json-utils_1.1.js

```diff
@@ -65,20 +65,22 @@ function convertDOMElToObject(entity) {
 STREET.utils.convertDOMElToObject = convertDOMElToObject;
 
 function getElementData(entity) {
-  if (!entity.isEntity) {
+  if (!entity.isEntity || entity.classList.contains('autocreated')) {
     return;
   }
   // node id's that should save without child nodes
   const skipChildrenNodes = ['environment', 'reference-layers'];
   const elementTree = getAttributes(entity);
   const children = entity.childNodes;
   if (children.length && !skipChildrenNodes.includes(elementTree.id)) {
-    elementTree['children'] = [];
+    const savedChildren = [];
     for (const child of children) {
       if (child.nodeType === Node.ELEMENT_NODE) {
-        elementTree['children'].push(getElementData(child));
+        const elementData = getElementData(child);
+        if (elementData) savedChildren.push(elementData);
       }
     }
+    if (savedChildren) elementTree['children'] = savedChildren;
```

Comment:
The condition is always true because empty list is true. You should write
`if (savedChildren.length > 0)` here.

Reply:
Yes, its my mistake, thanks for this!